pkg/services/ngalert/api/tooling/Makefile

```diff
@@ -11,6 +11,15 @@ PATH_UP = ../../../../..
 
 GENERATED_GO_MATCHERS = ./go/*.go
 
+SED_INPLACE := -i
+
+ifneq ($(OS),Windows_NT)
+	UNAME := $(shell uname)
+	ifeq ($(UNAME),Darwin)
+		SED_INPLACE = -i ''
+	endif
+endif
```

Comment:
@yuri-tceretian I requested your review because I know you use Windows for grafana dev. How do you usually run this makefile? I believe this should handle WSL and at least be somewhat tolerant to native windows (no `uname` on Windows, etc).

Do you think this is sufficient for Windows? I'd be happy to dig deeper if this doesn't cover all use cases, I want to make this work for everyone.

Reply:
I use Ubuntu linux to run make files. So we should be fine here.

Reply:
note that this would fail for someone running in mac but using the GNU `sed` by default. however, I don't have a better suggestion.